android/libpag/src/main/java/org/libpag/PAGPlayer.java

```diff
@@ -68,6 +68,18 @@ public void setSurface(PAGSurface surface) {
      */
     public native void setCacheEnabled(boolean value);
 
+    /**
+     * If set to true, PAG will save the rendering data to a file
+     * when first rendering BitmapComposition and VideoComposition,
+     * which will reduces memory consumption, and increases stability.
+     */
+    public native boolean useDiskCache();
```

Comment:
更新一下相关注释：If set to true, PAG will cache the associated rendering data into a disk file, such as the decoded image frames of video compositions. This can help reduce memory usage and improve device stability.